### PR TITLE
Allow GC Medigoron to be a hinted location

### PR DESCRIPTION
**Pull request by EntropicEidolon**

### CODE REVIEW

**Comment by ETR-BTF** [comment]
Was this location never added? lol

File: HintList.py [hunk 1]
```diff
@@ -487,6 +487,7 @@ def tokens_required_by_settings(world):
     'LH Underwater Item':                                          ("the #sunken treasure in a lake# is", None, 'exclude'),
     'Hideout Gerudo Membership Card':                              ("#rescuing captured carpenters# is rewarded with", None, 'exclude'),
     'Wasteland Bombchu Salesman':                                  ("a #carpet guru# sells", None, 'exclude'),
+    'GC Medigoron':                                                ("the #Medigoron# sells", None, 'exclude'),
```
Comment (ETR-BTF) [inline HintList.py:490]: Hate to be a pedant but I believe Medigoron is the NPC's name, so the "the" is not appropriate. It's also not used for Biggoron.

Comment (EntropicEidolon) [reply HintList.py]: Changed formatting to match Biggoron hint.

**Comment by r0bd0g** [comment]
Shouldn't you add one for carpet guy as well?

**Comment by fenhl** [comment]
That one already exists.

**Comment by r0bd0g** [comment]
Why did nobody tell whoever added that to also add one for medigoron?

**Comment by cjohnson57** [comment]
Yeah, seems was just an oversight.

**Review by cjohnson57 — APPROVED** [review]

**Pull request merged**